src/contrib/cluster/Akka.Cluster.Tools.Tests/PublishSubscribe/DistributedPubSubMediatorSpec.cs

```diff
@@ -54,5 +57,40 @@ public void DistributedPubSubMediator_can_be_activated_in_child_actor()
                 a.ShouldNotBe(ActorRefs.Nobody);
             });
         }
+
+        /// <summary>
+        /// Reproduction for https://github.com/akkadotnet/akka.net/issues/5352
+        /// </summary>
+        [Fact]
+        public async Task DistributedPubSubMediator_should_send_messages_to_dead_letter()
+        {
+            // arrange
+            var mediator = DistributedPubSub.Get(Sys).Mediator;
+            var actor = Sys.ActorOf((dsl, context) => { }, "childActor");
+
+            // act
+            // create a topic
+            mediator.Tell(new Subscribe("pub-sub", actor));
+            _ = ExpectMsg<SubscribeAck>();
+
+            // all subscribers should be removed from this topic
+            // topic actor will still be alive for default value of 120s
+            mediator.Tell(new Unsubscribe("pub-sub", actor));
+            _ = ExpectMsg<UnsubscribeAck>();
+
+            // assert
+            await EventFilter.DeadLetter<object>().ExpectAsync(1,
```

Comment:
Reproduction - as @eaba pointed out this spec would fail before because the TTL value kept the topic actor alive for 120s. The real bug was that the topic actor never published any `DeadLetter` instances if it was alive but had no subscribers.